drivers/soundwire/debugfs.c

```diff
@@ -85,10 +85,17 @@ static int sdw_slave_reg_show(struct seq_file *s_file, void *data)
 
```

Comment:
commit message: Adding missing SCP registers.

Reply:
ack done

Reply:
SCP registers needs to be updated to accomodate additional
register entries as per the SoundWire 1.2 specification.

Reply:
Updated the commit message with the soundwire spec version